content/download/linux.en.md

```diff
@@ -25,40 +25,44 @@ list of GRASS GIS packages.
 *  [Ubuntu](https://launchpad.net/~ubuntugis/+archive/ubuntu/ubuntugis-unstable/+packages?field.name_filter=grass) (ubuntugis-unstable)
 
-### <a name="GRASS-GIS-current"></a> GRASS GIS 7.8.6 (current)
+### <a name="GRASS-GIS-current"></a> GRASS GIS 8.0.0 (current)
 
 <div class="alert rounded-0 alert-success">
-<i class="fa fa-info-circle"></i> <u>Current stable release</u>, see <a href="https://trac.osgeo.org/grass/wiki/Grass7/NewFeatures78">GRASS GIS 7.8 new features</a> and <a href="https://trac.osgeo.org/grass/wiki/Release/7.8.6-News">GRASS GIS 7.8.6 announcement</a> for more information.</div>
+<i class="fa fa-info-circle"></i> <u>Current stable release</u>, see <a href="https://trac.osgeo.org/grass/wiki/Grass8/NewFeatures80">GRASS GIS 8.0 new features</a> and <a href="https://trac.osgeo.org/grass/wiki/Release/8.0.0-News">GRASS GIS 8.0.0 announcement</a> for more information.</div>
 
-*  [Generic 64bit](/grass78/binary/linux/snapshot) (weekly binary snapshot)
+*  [Generic 64bit](/grass80/binary/linux/snapshot) (weekly binary snapshot)
 *  [Gentoo](https://packages.gentoo.org/packages/sci-geosciences/grass)
-*  [EPEL7](https://copr.fedorainfracloud.org/coprs/neteler/grass78/) (RHEL7/Centos7/Scientific Linux7)
+*  [EPEL7](https://copr.fedorainfracloud.org/coprs/neteler/grass80/) (RHEL7/Centos7/Scientific Linux7)
 *  [EPEL8](https://src.fedoraproject.org/rpms/grass) (RHEL8/Centos8)
 *  [Fedora](https://src.fedoraproject.org/rpms/grass)
 *  [openSUSE](https://build.opensuse.org/package/show/Application:Geo/grass)
 *  [Ubuntu](https://launchpad.net/~ubuntugis/+archive/ubuntu/ubuntugis-unstable/+packages?field.name_filter=grass) (ubuntugis-unstable)
 
 <hr>
 
-### <a name="GRASS-GIS-old"></a> GRASS GIS 7.6.1 (legacy)
+### <a name="GRASS-GIS-old"></a> GRASS GIS 7.8.6 (legacy)
 
 <div class="alert rounded-0 alert-warning">
-<i class="fa fa-info-circle"></i> <u>Old stable release</u>, see <a href="https://trac.osgeo.org/grass/wiki/Grass7/NewFeatures76">GRASS GIS 7.6 new features</a> and  <a href="https://trac.osgeo.org/grass/wiki/Release/7.6.1-News">GRASS GIS 7.6.1 announcement</a> for more information.
+<i class="fa fa-info-circle"></i> <u>Old stable release</u>, see <a href="https://trac.osgeo.org/grass/wiki/Grass7/NewFeatures78">GRASS GIS 7.8 new features</a> and  <a href="https://trac.osgeo.org/grass/wiki/Release/7.8.6-News">GRASS GIS 7.8.6 announcement</a> for more information.
 </div>
 
-*  [Generic 64bit](/grass76/binary/linux/snapshot) (weekly binary snapshot)
-*  [EPEL7](https://copr.fedorainfracloud.org/coprs/neteler/grass76/) (RHEL7/Centos7/Scientific Linux7)
-*  [Fedora](https://copr.fedorainfracloud.org/coprs/neteler/grass76/)
+*  [Generic 64bit](/grass78/binary/linux/snapshot) (weekly binary snapshot)
+*  [Gentoo](https://packages.gentoo.org/packages/sci-geosciences/grass)
+*  [EPEL7](https://copr.fedorainfracloud.org/coprs/neteler/grass78/) (RHEL7/Centos7/Scientific Linux7)
```

Comment:
```suggestion
```
EPEL7 binaries do not exist due to lack of dependencies in EPEL7, see https://copr.fedorainfracloud.org/coprs/neteler/grass78/build/3218897/